ambry-admin/src/main/java/com.github.ambry.admin/AdminBlobStorageService.java

```diff
@@ -81,13 +78,14 @@
    * response handler controller and a router.
    * @param adminConfig the {@link AdminConfig} with configuration parameters.
    * @param adminMetrics the metrics instance to use in the form of {@link AdminMetrics}.
+   * @param clusterMap the {@link ClusterMap} to use.
```

Comment:
This part is a bit hasty. Could you make it something like "the {@link ClusterMap} that contains the source information of the cluster"?